grandcypher/__init__.py

```diff
@@ -180,6 +190,43 @@ def shortuuid(k=4) -> str:
     return "".join(random.choices(_ALPHABET, k=k))
 
 
+class SUBOP:
+    pass
+
+
+class SUBOP_EXIST(SUBOP):
+
+    def __call__(self, match: dict, executor: "GrandCypherExecutor"):
+        executor.clear_matches()
+        # backup some settings
+        bk_hints = executor._hints
+        bk_run_without_return = executor.run_without_return
+        bk_limit = executor._limit
+        bk_doublecheck_hint = executor._doublecheck_hint_result
+        # settings such that it favor EXISTS operation
+        executor.set_hints([match])
```

Comment:
Wow this `set_hints` turns out to be a super helpful API for this, I wasn't expecting that! :)